docs/source/getting_started/user_guide.md

```diff
@@ -40,27 +51,27 @@ class RandomClassificationDataset:
 
 There are a few parameters that need to be initialized before {class}`streaming.MDSWriter` gets called. Some of the parameters are optional, and others are required parameters. Let's look at each of them where we start with two required parameters.
 
-1. Provide the Local filesystem directory path to store the compressed dataset
+1. Provide the Local filesystem directory path to store the compressed dataset files.
     <!--pytest-codeblocks:cont-->
     ```python
     output_dir = 'test_output_dir'
     ```
 
-2. Provide the column field as `Dict[str, str]`, which maps a feature or label name with encoding type. All the supported encoding and decoding types can be found here.
+2. Provide the column field as `Dict[str, str]`, which maps a feature name or label name with a streaming supported encoding type.
     <!--pytest-codeblocks:cont-->
     ```python
     fields = {'x': 'pkl', 'y': 'pkl'}
     ```
 
 The below parameters are optional to {class}`streaming.MDSWriter`. Let's look at each one of them
 
-1. Provide a name of a compression algorithm; the default is `None`. We support families of compression algorithms, and all of them can be found here.
+1. Provide a name of a compression algorithm; the default is `None`. Streaming support families of compression algorithms such as `br`, `gzip`, `snappy`, `zstd`, and `bz2` with the level of compression.
```

Comment:
Streaming supports

Reply:
Done in https://github.com/mosaicml/streaming/pull/33/commits/a634dbdbd8039c0755687d3cc8172cba15c977ed.